sw/device/tests/penetrationtests/firmware/ibex_fi.h

```diff
@@ -169,6 +169,25 @@ status_t handle_ibex_fi_char_sram_static(ujson_t *uj);
  */
 status_t handle_ibex_fi_char_sram_read(ujson_t *uj);
 
+/**
+ * ibex.fi.char.sram_write_static command handler.
+ *
+ * This FI penetration tests executes the following instructions:
+ * - Set the trigger.
+ * - Add 10 NOPs to delay the trigger
+ * - Write 32 static values into SRAM.
```

Comment:
are there 10 NOPs I am missing? and aren't you writing 64 values?

Reply:
Ah thanks, yes. I've now added the nops and updated the description.